src/helpers/api/getExtrasApi.ts

```diff
@@ -0,0 +1,7 @@
+import { getCurrentNetwork } from './getCurrentNetwork';
+
+export function getExtrasApi() {
+  const { extrasApi } = getCurrentNetwork();
```

Comment:
Is this helper really needed? All it does is call another function and returns that data.

Reply:
It is needed

Reply:
Got it. 👍